autotst/geometry.py

```diff
@@ -102,13 +102,20 @@ def __init__(
             atom_indices,
             dihedral,
             mask,
-            reaction_center=False):
+            angles,
+            description = 'hindered',
+            reaction_center = False,
+            center_atoms = []):
         self.index = index
         self.atom_indices = atom_indices
         self.dihedral = dihedral
         self.mask = mask
+        self.angles = angles
```

Comment:
If a torsion is made up of atoms i, j, k, and l, `self.angles` are the angles between atoms i, j, and k and atoms j, k, and l. Correct? This feels like redundant information considering `Angle` objects should exist with this information.

Reply:
I agree, we don't really need this.  It's also the reason why the unit tests failed.  I removed it.